bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/AccountHandler.java

```diff
@@ -770,8 +791,7 @@ void handleWebsocketCommand(JsonPushCommand pushCommand) {
                     break;
                 default:
                     String payload = pushCommand.payload;
-                    if (payload != null && StringUtils.isNotEmpty(payload) && payload.startsWith("{")
-                            && payload.endsWith("}")) {
+                    if (payload != null && !payload.isEmpty() && payload.startsWith("{") && payload.endsWith("}")) {
```

Comment:
I think the `isEmpty`-check is useless. if it's not `null` and `startswith` something, it can't be empty.

Reply:
Yes, your assumption is correct.